File: Question_9/Java/Question9.java
import java.io.*;
import java.lang.reflect.Array;
import java.util.*;

class Solution {

    // Complete the organizingContainers function below.
    static String organizingContainers(int[][] container) {
        long[] rows = new long[container.length];
        long[] cols = new long[container.length];
        for(int i = 0; i < container.length; i++){
            for(int j = 0; j < container.length; j++){
                rows[i] += container[i][j];
                cols[j] += container[i][j];
            }
        }
        Arrays.sort(rows);
        Arrays.sort(cols);
        if(Arrays.equals(rows, cols)) return "Possible";
        return "Impossible";
    }

    private static final Scanner scanner = new Scanner(System.in);

    public static void main(String[] args) throws IOException {

        Scanner in = new Scanner(System.in);
        int q = in.nextInt();
        String answer="";
        while(q-->0)
        {
            int n = in.nextInt();
            int[][] M = new int[n][n];
            for(int i=0; i < n; i++)
                for(int j=0; j < n; j++)
                    M[i][j] = in.nextInt();
            String result= organizingContainers(M);
            if (answer.equals(""))
            {
                answer= result;
            }
            else
            {
                answer= answer + ","+result;
            }
        }

        // Do not remove below line
        System.out.println(answer);
        // Do not print anything after this line

        scanner.close();
    }
}

File: Question_9/Python/Question9.py
def organizingContainers(container):
    rows = [0] * len(container)
    cols = [0] * len(container)
    for i in range(len(container)):
        for j in range(len(container)):
            rows[i] += container[i][j]
            cols[j] += container[i][j]
    if sorted(rows) == sorted(cols):
        return "Possible"
    return "Impossible"


if __name__ == "__main__":
    q = int(input().strip())
    answer = ""
    for a0 in range(q):
        n = int(input().strip())
        container = []
        for container_i in range(n):
            container_t = [int(container_temp) for container_temp in input().strip().split(' ')]
            container.append(container_t)
        result = organizingContainers(container)
        if (answer == ""):
            answer = str(result)
        else:
            answer = answer + "," + str(result)
    # Do not remove below line
    print(answer)
    # Do not print anything after this line
